compiler/bin-js_of_ocaml/compile.ml

```diff
@@ -285,7 +285,7 @@ let run
   | `None ->
       let prims = Linker.list_all () |> StringSet.elements in
       assert (List.length prims > 0);
-      let code, uinfo = Parse_bytecode.predefined_exceptions () in
+      let code, uinfo = Parse_bytecode.predefined_exceptions ~target:`JavaScript in
```

Comment:
Why do we pass `~target` explicitly here instead of looking at `Config.target` ? Is `Parse_bytecode` different from other modules ? If we end up deciding to keep it this way, we could define/bind target at the top of this file

Reply:
This was an oversight on my part. Fixed.